src/pages/APIPage.tsx

```diff
@@ -96,7 +96,7 @@ export const APIPage: FunctionComponent = () => {
                   <ApiDoc apiContent={apiState.api} groupedOperations={groupedOperations.value} /> }
             </SidebarContent>
           </Sidebar>
-          <BackToTop  style={{position: 'fixed'}}/>
+          <BackToTop className='pf-c-back-to-top'/>
```

Comment:
The class name `pf-c-back-to-top` is the default one for that component (as documented [here](https://patternfly-react.surge.sh/components/back-to-top)), so this is not required.

```suggestion
          <BackToTop/>
```

Reply:
Ah okay, so class name is only necessary when its different from the default one?

Reply:
> Ah okay, so class name is only necessary when its different from the default one?

Yes, if you inspect the code that class name `pf-c-back-to-top` is already part of the div element. We could have used a custom name but I think it's more inline with what we have already in the same css file.